tfjs-backend-webgpu/src/device_limits.ts

```diff
@@ -15,12 +15,13 @@
  * =============================================================================
  */
 
-export const MAX_COMPUTE_WORKGROUP_INVOCATIONS = 256;
+export let maxComputeWorkgroupsPerDimension = 65535;
+export let maxComputeWorkgroupStorageSize = 16352;
+export let maxStorageBufferBindingSize = 134217728;
```

Comment:
Maybe remove maxStorageBufferBindingSize? It's not possible to exceed this value?

Reply:
I don't like the idea here to put these as global variables and create another file for them. They can be a new deviceLimits object in WebGPUBackend. In index.ts, can we put all the creation of adapter and device into WebGPUBackend init?
In addition, we can put them in a dict so that we can iterate them to initialize. The initial values here are not necessary.